yt/frontends/swift/io.py

```diff
@@ -0,0 +1,168 @@
+"""
+Swift data-file handling function
+
+
+
+
+"""
+from __future__ import print_function
```

Comment:
the future imports aren't necessary since you're not using print anywhere

Reply:
this is a copy and paste, thanks for the catch :)